dlls/ff/ff_player.cpp

```diff
@@ -8024,12 +8024,12 @@ void CFFPlayer::RemoveMeFromKillAssists( )
 	}
 }	
 
-float CFFPlayer::GetJetpackFuel()
+int CFFPlayer::GetJetpackFuel()
 {
-	return m_flJetpackFuel;
+	return m_iJetpackFuel;
 }
 
-void CFFPlayer::SetJetpackFuel(float newFuel)
+void CFFPlayer::SetJetpackFuel(int newFuel)
 {
-	m_flJetpackFuel = max(100.0f, newFuel);
+	m_iJetpackFuel = max(200, newFuel);

```

Comment:
Note: This value will need to get updated in the base_teamplay Lua as well (https://github.com/fortressforever/fortressforever-scripts/pull/64). The alternative would be to change this function to SetJetpackFuelPercent (either by name or just in terms of functionality), so that `SetJetpackFuel(100)` always sets fuel to 100% regardless of the internal representation. Doing that might be better?